lib/seccomp/CSystemCallFilter_Linux.cc

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+#include "seccomp/CSystemCallFilter.h"
+
+#include <core/CLogger.h>
+
+#include <linux/audit.h>
+#include <linux/filter.h>
+#include <sys/prctl.h>
+#include <sys/syscall.h>
+
+
+namespace ml {
+namespace seccomp {
+
```

Comment:
Everything from here down to the end of `canUseSeccompBpf()` should be in an unnamed namespace to enforce internal linkage.